src/main/resources/templates/methodNotification.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <script>
+		var getOrigin = window.location;
+		window.parent.postMessage("3DS.Notification.Received", getOrigin.origin);
```

Comment:
Idk, what about this?
```suggestion
		var getOrigin = window.location.origin;
		window.parent.postMessage("3DS.Notification.Received", getOrigin);
```

Reply:
yess, more clean